.github/ISSUE_TEMPLATE/direct-from-marketing.yml

```diff
@@ -0,0 +1,43 @@
+name: Add marketing blog content
+description: >
+  For adding marketing-approved content to the DevBlog.
+labels: ["content","developer blog"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Let's get some content up!
+        
+  - type: input
+    id: contact
+    attributes:
+      label: Contact Details
+      description: How can we get in touch with you if we need more info?
+      placeholder: your preferred email and/or dbt Slack handle
+    validations:
+      required: true
+
+  - type: checkboxes
+    id: from-discussion
+    attributes:
+      label: This issue is being made to create a marketing-approved piece.
+      description: >
+        This content will typically fall under glossary pages, community recap events, and devblog content for campaigns. Please reach out to John Blust if there's any confusion here.
+      options:
+        - label: I am directly creating this issue for a marketing-approved piece.
+    validations:
+      required: true
+
+  - type: textarea
+    attributes:
+      label: Link to the marketing artifact/source material you are referencing.
+      description: Please link to the artifact (content brief/event recap) you are using to guide your work.
+    validations:
+      required: false
+      
+  - type: textarea
+    attributes:
+      label: Link to the initial outline we will develop through this Issue.
+      description: Please link to the initial outline in FigJam, Notion, or Google Docs
+    validations:
+      required: true
```

Comment:
do you want this to be required? i'm just wondering if you'll always have a public facing outline necessarily for these

Reply:
Maybe you even just merge this one into the one above it for this type?

Reply:
I think I'm just going to merge into one section. For the most part, our outlines here should be public facing and we've been putting our outlines in the Issues so far. Will change this in the future if we're feeling it's no longer appropriate!

Reply:
Ok made some minor changes, thanks for your review!